webpack.config.js

```diff
@@ -160,7 +161,10 @@ if ( calypsoEnv === 'desktop' ) {
 
 const jsLoader = {
 	test: /\.jsx?$/,
-	exclude: /node_modules/,
+	include: [
+		path.resolve( './client' ),
+		fs.realpathSync( './node_modules/notifications-panel' )
```

Comment:
that's the idea. pull the original source in and let Calypso's build process make it instead of loading an already minified bundle. share the dependencies.

we may not need to do it this way if we can safely share dependencies, but I want to keep versions pinned in both places and not worry about keeping those in sync or pulling in two copies of everything.